budgetkey_data_pipelines/pipelines/entities/moital/pipeline-spec.yaml

```diff
@@ -0,0 +1,14 @@
+moital_service_providers:
+  pipeline:
+    - run: add_metadata
+      parameters:
+        name: moital_service_providers
+        title: Moital Service Providers
+    - run: scraper
```

Comment:
Did you saw that the ServiceContractors page have one extra column?
In other words the list will contain empty values in the ManpowerContractors output.